src/GafferSceneUI/VisualiserTool.cpp

```diff
@@ -108,6 +108,11 @@ float const g_vectorScaleInc = 0.01f;
 
 const Color3f g_vectorColorDefault( 1.f, 1.f, 1.f );
 
+// Orientation constants
+const Color3f g_colorX( 1.f, 0.f, 0.f );
+const Color3f g_colorY( 0.f, 1.f, 0.f );
+const Color3f g_colorZ( 0.f, 0.f, 1.f );
```

Comment:
We pass these constants into a struct in C++, which goes into a uniform buffer in OpenGL, which eventually ends up in an array of colours in GLSL. But couldn't the GLSL just do something like this?

```
vec3 color = 0.0;
color[axis] = 1.0;
```

Alternatively, if we're going to use constants, then I quite like the slightly less garish ones we use for the transform handles (where blue also shows up better than `0, 0, 1`). And we could presumably declare the constants in the GLSL source rather than passing them all the way through.

Reply:
I put in the more subtle colors in https://github.com/GafferHQ/gaffer/pull/6368/commits/e7b958a37a45c9b28ecbc406fdc5ad8d082c74bb and hard coded them in the shader as well. That seems to look pretty nice now.